libbindgen/tests/headers/const_bool.hpp

```diff
@@ -0,0 +1,4 @@
+const bool k = true;
```

Comment:
You need to make the first line of this test `// bindgen-unstable`, given this is not available in LLVM 3.8.